uint8 GO_IN_FRONT = 0
uint8 GO_NEXT = 1

uint8 move_type
string frame_id
---
int32 SUCCESS = 0
int32 NO_TABLE_TO_OBJ_TRANSFORM = -1
int32 NO_TABLE_TO_FOOTPRINT_TRANSFORM = -2
int32 NO_MAP_TO_TABLE_TRANSFORM = -3
int32 PREMPTED = -4


int32 error_code
time action_end
---
float32 distance_to_goal
time action_start
